src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      data-qa="big"
+      action="#"
+      method="get"
+    >
+      <input
+        class="search-input input--big"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form
+      class="search-form"
+      data-qa="small"
+      action="#"
+      method="get"
+    >
+      <input
+        class="search-input input--small"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
   </body>
+
 </html>
```

Comment:
```suggestion
  </body>
</html>
```
Don't leave empty lines between parent and child elements